www/src/components/guidelines/color/modal.js

```diff
@@ -1,57 +1,66 @@
+/** @jsx jsx */
+import { jsx } from "theme-ui"
 import React from "react"
-import styled from "@emotion/styled"
-import themeGet from "@styled-system/theme-get"
-
-import { Box, Flex, Heading, Text } from "../system"
+import { Heading, Text } from "../system"
+import { Box, Flex } from "theme-ui"
 import {
   getA11yLabel,
   getTextColor,
   a11y,
 } from "../../../utils/guidelines/color"
 import { focusStyle } from "../../../utils/styles"
+import { mediaQueries } from "gatsby-design-tokens/dist/theme-gatsbyjs-org"
 import { colors as themeColors } from "gatsby-design-tokens/dist/theme-gatsbyjs-org"
 
-const Column = styled(Flex)()
-
-Column.defaultProps = {
-  flexWrap: `wrap`,
-  width: { lg: `50%` },
-  p: { xxs: 6, md: 8 },
-  m: 0,
-}
+const Column = ({ children, ...rest }) => (
+  <Flex
+    sx={{
+      flexWrap: `wrap`,
+      width: `100%`,
+      p: 6,
+      m: 0,
+      ...rest,
+      [mediaQueries.md]: {
+        p: 8,
+      },
+      [mediaQueries.lg]: {
+        width: `50%`,
+      },
+    }}
+  >
+    {children}
+  </Flex>
+)
 
-const CloseButton = styled(Flex)(
-  {
-    border: 0,
-    cursor: `pointer`,
-    outline: 0,
-    WebkitAppearance: `none`,
-  },
-  props => {
-    return {
-      transition: `all ${themeGet(`transition.speed.fast`)} ${themeGet(
-        `transition.curve.default`
-      )}`,
+const CloseButton = ({ children, ...rest }) => (
+  <button
+    {...rest}
+    sx={{
+      alignItems: `center`,
+      bg: `transparent`,
+      borderRadius: 6,
+      fontSize: 6,
+      width: 48,
+      height: 48,
+      justifyContent: `center`,
+      ml: `auto`,
+      border: 0,
+      cursor: `pointer`,
+      outline: 0,
+      WebkitAppearance: `none`,
+      pb: 6,
+      transition: t =>
+        `all ${t.transition.speed.fast} ${t.transition.curve.default}`,
       ":hover, :focus": {
         ...focusStyle,
-        color: themeGet(`colors.orange.50`)(props),
+        color: t => t.colors.orange[50],
```

Comment:
```suggestion
        color: `orange.50`,
```

No need to grab this via `t`/[functional values](https://theme-ui.com/sx-prop/#functional-values) here as the CSS `color` prop is mapped to the `theme-ui` `colors` scale directly.